File: Decorator/Java/src/decoratorSandwich/fool.java
package decoratorSandwich;

public class fool extends SandwichDecorator {

    public fool(Sandwich sandwich) {
        super(sandwich);
    }

    @Override
    public double getcost() {
        return super.getcost() +2.0;
    }

    @Override
    public String getDescription() {
        return super.getDescription() + "  fool";
    }
}

File: Decorator/Python/Fool.py
# -*- coding: utf-8 -*-
"""
Created on Thu May 19 19:02:10 2022

@author: Mahmoud Saeed
"""

import SandwichDecorator

class Fool(SandwichDecorator.SandwichDecorator):
    
    def __init__(self, sandwich):
         super().__init__(sandwich)
        
    def getCost(self):
        return super().getCost() + 2.0 
    
    def getDesc(self):
        return super().getDesc() + " + fool "